reana_db/utils.py

```diff
@@ -299,6 +299,9 @@ def update_users_disk_quota(
 ) -> None:
     """Update users disk quota usage.
 
+    User disk quotas will be calculated from workflow disk quotas,
+    so the latter should be updated before the former.
```

Comment:
Cosmetics: perhaps add the explicit function name for clarity, e.g.  "User disk quota usage will be calculated from the individual workflow disk quota usage numbers, so this function should be typically called only after `update_workflows_disk_quota()`."

Reply:
Please also write an entry into `CHANGES.rst` about performance improvements for disk quota calculations. (The last PR also omitted news item about this.)